docs/layouts/shortcodes/literalInclude.html

```diff
@@ -43,6 +43,7 @@
       segments will be separated by an ellipsis to indicate elided code.
   - gitHub: specify GitHub link to generate a link to example. E.g
     ``https://github.com/osohq/gitclub/``
+  - ellipsis: override the default marker for elided code ("...").
```

Comment:
Add `hl_lines` up here, and write that it only works with Polar?

Reply:
d116c43